graph/Makefile

```diff
@@ -1,11 +1,14 @@
 .DEFAULT_GOAL := help
+export FLASK_APP=src/main.py
+export FLASK_RUN_PORT=8000
+export FLASK_RUN_HOST=0.0.0.0
```

Comment:
Is this deliberate as in: https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html ?